code/game/gamemodes/colonialmarines/halloween2016.dm

```diff
@@ -507,7 +507,7 @@
 				H.equip_to_slot_or_del(new /obj/item/clothing/under/marine/veteran/PMC/leader(H), SLOT_W_UNIFORM)
 				H.equip_to_slot_or_del(new /obj/item/clothing/suit/storage/marine/veteran/PMC/leader(H), SLOT_WEAR_SUIT)
 				H.equip_to_slot_or_del(new /obj/item/clothing/head/helmet/marine/veteran/PMC/leader(H), SLOT_HEAD)
-				H.equip_to_slot_or_del(new /obj/item/clothing/mask/gas/PMC/leader(H), SLOT_WEAR_MASK)
+				H.equip_to_slot_or_del(new /obj/item/clothing/mask/gas/marine/PMC/leader(H), SLOT_WEAR_MASK)
```

Comment:
Could you nuke this?

Reply:
As in, removing the commented code?

Reply:
Well, the whole game mode, I guess, but that may be better in a different PR.